Gruntfile.js

```diff
@@ -31,6 +33,21 @@ module.exports = function Gruntfile( grunt ) {
 				]
 			}
 		},
+		jsdoc: {
+			dist: {
+				src: [ 'src/*.js', 'test/*.js' ],
+				options: {
+					encoding: 'utf8',
+					readme: 'README.md',
+					tutorials: 'help',
+					recurse: true,
+					template: 'node_modules/jsdoc-wmf-theme',
+					destination: 'doc',
+					'package': 'package.json',
+					ignoreWarnings: false
```

Comment:
Might want to add the markdown and plugin? Maybe the hierarchy one if this is going to get complicated 😜

https://github.com/wikimedia/oojs/blob/master/.jsdoc.json#L14

Reply:
Done.